src/websocket.c

```diff
@@ -263,16 +266,18 @@ callback_notify(struct lws *wsi, enum lws_callback_reasons reason, void *user, v
       case LWS_CALLBACK_ESTABLISHED:
 	// Initialize session data for new connections
 	memset(session_data, 0, sizeof(struct ws_session_data_notify));
+	session_data->counter = write_events_counter; // set it -1 so we can still send during this cycle
```

Comment:
Sorry, one thing left ... can you remove the comment. It is now misleading.

Reply:
Sorry forgot to do that..done.